legacy-elements/bo-switch-config/index.html

```diff
@@ -14,14 +14,29 @@
-      ></bo-switch-config>
+<!--      <bo-switch-config-->
+<!--        id="switch-component-config"-->
+<!--        dictionary='{"cancel": "annuler", "properties": "propriétés"}'-->
+<!--        from='{"name":"Input","options":{-->
+<!--        "hidden":{"type":"expression","value":"toto","label":"Hidden"},-->
+<!--        "pageSize":{"type":"constant","value":"10","label":"Page Size"},-->
+<!--        "content":{"type":"expression","value":"json_data","label":"Content"},-->
+<!--        "labelHidden":{"type":"constant","value":false,"label":"Label hidden"}-->
+<!--        }}'-->
+<!--        to='{"id":"CustomInput","name":"CustomInput","options":{-->
+<!--        "hidden":{"label":"Hidden","name":"hidden","type":"boolean","defaultValue":false,"bond":"expression"},-->
+<!--        "pageSize":{"value":"","label":"Page Size", "bond": "variable"},-->
+<!--        "content":{"bond":"expression","value":"json_data","label":"Content"},-->
+<!--        "labelHidden":{"label":"Label hidden","name":"labelHidden","type":"boolean","defaultValue":false,"bond":"constant"}-->
+<!--        }}'-->
+<!--      ></bo-switch-config>-->
+      <bo-switch-config id="switch-component-config" dictionary="{}"
```

Comment:
Please split the lines for better readability.
(as you did in the commented lines just above, that can be removed BTW)